cves/kernel/CVE-2016-0823.yml

```diff
@@ -468,12 +487,10 @@ CWE_instructions: |
     CWE: 123            # also ok
 CWE:
 - 200
-CWE_note: |
-  CWE as registered in the NVD. If you are curating, check that this
-  is correct and replace this comment with "Manually confirmed".
+CWE_note: Manually confirmed
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
+nickname: Hammer Time
 CVSS:
```

Comment:
I saw that the NVD was going to reanalyze this vulnerability but this is what they have for now for CVSS: 3.0/AV:L/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N